lib/src/file_picker_web.dart

```diff
@@ -108,15 +108,34 @@ class FilePickerWeb extends FilePicker {
       });
     }
 
+    void cancelledEventListener(_) {
+      window.removeEventListener('focus', cancelledEventListener);
+
+      // This listener is called before the input changed event,
+      // and the `uploadInput.files` value is still null
+      // Wait for results from js to dart
+      Future.delayed(Duration(milliseconds: 500)).then((value) {
```

Comment:
Can we use a microtask here instead? (`scheduleMicrotask(callback)`)?

Reply:
Thank you

Tested with `scheduleMicrotask(callback)` and it doesn't work: the `callback` is called too soon